src/Routes/SessionRoute/SessionRoute.js

```diff
@@ -205,7 +225,11 @@ componentDidMount() {
               className='button green-button'
               onClick={() => this.toggleTargetComplete(student.studentGoalId, student.iscomplete)
                 }>Target Complete</button>}
-            <p>{student.subgoal ? student.subgoal : this.state.learningTarget}</p>
+
+              <p>{this.state.learningTarget}</p>
+       
+            <p>{student.studentSubgoal ? student.studentSubgoal : ''}</p>
```

Comment:
Will this display the same goal twice if the learning target and the student subgoal are the same?